web/renderer/components/DatabaseHeaderAndNav/queries.ts

```diff
@@ -11,3 +11,36 @@ export const RESET_DATABASE = gql`
     resetDatabase(newDatabase: $newDatabase)
   }
 `;
+
+export const CURRENT_CONNECTION = gql`
+  query CurrentConnection {
+    currentConnection {
+      connectionUrl
+      name
+      hideDoltFeatures
+      useSSL
+      type
+      isDolt
+    }
+  }
+`;
```

Comment:
Is this used? Seems like you use the StoredConnections query instead?

Reply:
this is for displaying the current connection name in the breadcrumbs:
![Screenshot 2024-11-20 at 4 54 43 PM](https://github.com/user-attachments/assets/97000253-20f5-4e75-87b5-ebf41dd03f7e)